containers.go

```diff
@@ -312,22 +312,30 @@ func runContainer(daemon *docker.Client, id string, logger log15.Logger, logfile
 	// If we're the outer shell, log straight to stderr, nothing fancy
 	stdout := io.Writer(os.Stdout)
 	stream := io.Writer(os.Stderr)
+	var log, hookedR, hookedW *os.File
+	var fdsToClose []io.Closer
+	var err error
```

Comment:
Declaring `err` here just makes the code convoluted. It "hints" that errors are propagated through for some reason. However the code below just checks for error conditions and either returns or continues. There's no need to declare it up front.